frontend-cert/js-projects/lab-factorial/user-stories.md

```diff
@@ -0,0 +1,8 @@
+1. You should ask the user for a number using a prompt window and store the number in a variable called `inputNumber`.
+   
+2. Declare a variable `result` and assign it the value of 1.
+   
+3. Using a for loop, you should loop through all numbers from `1` to `inputNumber` (inclusive) and for each number, multiply the `result` variable by the current number.
```

Comment:
```suggestion
3. Using a for loop, you should loop through all numbers from `1` to `inputNumber` (inclusive) and for each number, multiply the `result` variable by the current number and assign the result to the `result` variable.
```
It may not be necessary, though.